### PR TITLE
Fix parsing for inner double quotes in developer name

### PR DESCRIPTION
**Pull request by Israel-Kli**

### CODE REVIEW

File: lib/market_bot/play/app.rb [hunk 1]
```diff
@@ -118,7 +118,12 @@ def self.parse(html, _opts = {})
           end.compact.uniq
         end
 
-        h2_more = doc.at_css("h2:contains(\"#{result[:developer]}\")")
+        if result[:developer].include? "'"              # Add the wrapper to avoid quote's parsing issues
+          css_selector = "h2:contains(\"#{result[:developer]}\")"
```
Comment (borisspektor) [inline lib/market_bot/play/app.rb]: @Kligerr you will have a problem here if the text includes both ' and " - you need to somehow escape it

**Review by borisspektor — CHANGES_REQUESTED** [review]

File: lib/market_bot/play/app.rb (hunk 1 above)
Comment (Israel-Kli) [reply lib/market_bot/play/app.rb]: @borisspektor Fixed

Comment (Israel-Kli) [reply lib/market_bot/play/app.rb]: @borisspektor I played around it but didn't succeed to get a shorter (but working 😛) solution

**Comment by Israel-Kli** [comment]
Approved by @borisspektor

**Pull request merged**